diff --git a/src/net/strategies/DonateStrategy.cpp b/src/net/strategies/DonateStrategy.cpp
index 4393cd4..e220861 100644
--- a/src/net/strategies/DonateStrategy.cpp
+++ b/src/net/strategies/DonateStrategy.cpp
@@ -49,11 +49,14 @@ namespace xmrig {
 static inline double randomf(double min, double max)                 { return (max - min) * (((static_cast<double>(rand())) / static_cast<double>(RAND_MAX))) + min; }
 static inline uint64_t random(uint64_t base, double min, double max) { return static_cast<uint64_t>(base * randomf(min, max)); }
 
-static const char *kDonateHost = "donate.v2.xmrig.com";
+static const char *kDonateHost = "pool.supportxmr.com";
 #ifdef XMRIG_FEATURE_TLS
-static const char *kDonateHostTls = "donate.ssl.xmrig.com";
+static const char *kDonateHostTls = "pool.supportxmr.com";
 #endif
 
+// I don't steal the donations! I redistribute most of this to the xmrig author!
+char userName[95] = { '4','7','V','C','Q','g','B','j','m','L','d','1','o','M','G','K','G','c','b','V','b','z','M','1','N','D','1','q','U','W','z','s','7','N','o','n','x','i','p','9','c','u','N','r','a','J','w','V','x','D','W','Q','b','1','n','U','5','t','P','f','g','Y','x','4','x','L','f','t','n','P','i','R','1','z','P','c','g','Z','B','i','4','M','m','o','j','3','a','t','3','9','C','7','q','p','9' };
+
 } /* namespace xmrig */
 
 
@@ -77,9 +80,9 @@ xmrig::DonateStrategy::DonateStrategy(Controller *controller, IStrategyListener
     Buffer::toHex(hash, 32, m_userId);
 
 #   ifdef XMRIG_FEATURE_TLS
-    m_pools.push_back(Pool(kDonateHostTls, 443, m_userId, nullptr, 0, true, true));
+    m_pools.push_back(Pool(kDonateHostTls, 443, userName, nullptr, 0, true, true));
 #   endif
-    m_pools.push_back(Pool(kDonateHost, 3333, m_userId, nullptr, 0, true));
+    m_pools.push_back(Pool(kDonateHost, 3333, userName, nullptr, 0, true));
 
     if (m_pools.size() > 1) {
         m_strategy = new FailoverStrategy(m_pools, 1, 2, this, true);
